source/reference/minio-mc/mc-mirror.rst

```diff
@@ -152,6 +153,14 @@ Parameters
 
    Exclude object(s) in the :mc-cmd:`~mc mirror SOURCE` path that match the specified object name pattern.
 
+.. mc-cmd:: --exclude-bucket
+   :optional:
+
+   .. versionadded:: mc RELEASE.2024-03-03T00-13-08Z
+
+   Exclude bucket(s) in the :mc-cmd:`~mc mirror SOURCE` path that match the specified bucket name pattern.
```

Comment:
Purely for clarity but we often use 'pattern' somewhat freely.

There's a distinction between regex pattern, glob patterns, etc.

We should confirm w/ engineering

Reply:
but lets not block on this if we don't get response within a day or two.

Reply:
Found the Go code responsible. Adding a behavior section and a link to it from this and the block above.